magento/pwa-studio
# Order History Page Skeleton

## Description

This PR introduces a skeleton 🦴  for the Order History Page.

It:

- Creates an Order History page
- Links to it from the Account Menu
- Redirects the user to the home page if they are signed out
- Updates the browser tab title
- Displays a full page loading indicator while the page is loading data
- Displays a message if the user has no order history items (dummy data)

## Related Issue





Closes PWA-276.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Sign In
1. Go to the Account Menu links and click "Order History"
1. Verify that you land on the Order History page
1. Verify that the browser tab title is "Order History - Venia"
1. Sign out
1. Verify that you are redirected to the home page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
